specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1480,10 +1482,10 @@ components:
       properties:
         self:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink"
-        bagAdressen:
+        Adressen:
           title: "betreft"
           type: "array"
-          description: "Dit een koppeling naar het Adres zoals dat in de HC-Bag API wordt aangeboden."
+          description: "Dit is een link naar het adres vande objectlocatie van de Kadatsraal Onreorende Zaak."
```

Comment:
"Dit is een link naar het adres vande objectlocatie van de Kadatsraal Onreorende Zaak."
--->
"Dit is een link naar het adres van de objectlocatie van de Kadastraal Onroerende Zaak."